tests/validation/lib/aws.py

```diff
@@ -144,19 +144,24 @@ def create_node(self, node_name, ami=AWS_AMI, ssh_user=AWS_USER,
 
         if wait_for_ready:
             node = self.wait_for_node_state(node)
-            node.ready_node()
+            if public_ip:
+                node.ready_node()
+            else:
+                time.sleep(60)
```

Comment:
Sleep in both places is because there is no ssh access, so no way to really validate `ready_node()` other than wait for a bit. I noticed that without the sleep there was some flakiness in the initial connections to the nodes. It worked most of the time at 30 seconds but I saw 2 failures in my tests when only having the sleep for 30 seconds so did it for 60 just to be safe.